src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -33,6 +33,18 @@
 			class="new-message">
 			<form
 				class="new-message-form">
+				<div
+					class="new-message-form__button">
+					<EmojiPicker @select="addEmoji">
+						<button
+							type="button"
+							class="new-message-form__icon new-message-form__button"
+							:aria-label="t('spreed', 'Add emoji')"
+							:aria-haspopup="true">
+							🙂
```

Comment:
As per screenshots in the design issue I would propose to use the material icon here (instead of yellow colored emoji):
```js
import EmoticonOutline from 'vue-material-design-icons/EmoticonOutline'
```
```suggestion
							<EmoticonOutline
							     :size="24">
```